geopandas/plotting.py

```diff
@@ -301,6 +303,13 @@ def plot_series(s, cmap=None, color=None, ax=None, figsize=None, **style_kwds):
     figsize : pair of floats (default None)
         Size of the resulting matplotlib.figure.Figure. If the argument
         ax is given explicitly, figsize is ignored.
+    set_aspect : 'auto', 'equal' or float (default 'auto')
```

Comment:
Should the keyword be just "aspect" instead of "set_aspect"? (like we could have a "title" or "xlabel" keyword which translates to matplotlib's set_title / set_xlabel)